tests/child-dispatch.test.sh

```
@@ -1222,4 +1222,76 @@ PACKAGE_LIB="$TMP/installed-package/lib/child-dispatch.sh" PACKAGE_HANDOFF="$TMP
+[ "$(uberdev_child_validate_finding_verifier_result "$VERIFY_DIR/result.md")" = "100 reproduced-from-diff" ]
+# Every controller-assigned reason token is accepted by the grammar too: the
+# controller writes the same document shape when no child opinion exists.
+for verifier_reason in pre-existing out-of-scope-line linter-domain gate-disabled over-cap-unverified verifier-unavailable; do
+  verifier_result 50 "$verifier_reason"
+  [ "$(uberdev_child_validate_finding_verifier_result "$VERIFY_DIR/result.md")" = "50 $verifier_reason" ]
+done
+
+# Out-of-range, non-integer, quoted, and leading-zero scores are refused.
+for bad_score in 101 -1 1000 abc '"80"' 007 80.0 ''; do
+  verifier_result "$bad_score" reproduced-from-diff
+  ! uberdev_child_validate_finding_verifier_result "$VERIFY_DIR/result.md" >/dev/null 2>&1
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)